### ISSUE TITLE
Error loading model on 0.2.8-rc1-rocm

### ISSUE THREAD

**Issue by dvelez3815** [ollama/ollama#5850]
### What is the issue?

segmentation fault (core dumped)  error

```
2024/07/22 13:17:50 routes.go:1100: INFO server config env="map[CUDA_VISIBLE_DEVICES: GPU_DEVICE_ORDINAL: HIP_VISIBLE_DEVICES: HSA_OVERRIDE_GFX_VERSION: OLLAMA_DEBUG:false OLLAMA_FLASH_ATTENTION:false OLLAMA_HOST:http://0.0.0.0:11434 OLLAMA_INTEL_GPU:false OLLAMA_KEEP_ALIVE:5m0s OLLAMA_LLM_LIBRARY: OLLAMA_MAX_LOADED_MODELS:0 OLLAMA_MAX_QUEUE:512 OLLAMA_MAX_VRAM:0 OLLAMA_MODELS:/root/.ollama/models OLLAMA_NOHISTORY:false OLLAMA_NOPRUNE:false OLLAMA_NUM_PARALLEL:0 OLLAMA_ORIGINS:[http://localhost https://localhost http://localhost:* https://localhost:* http://127.0.0.1 https://127.0.0.1 http://127.0.0.1:* https://127.0.0.1:* http://0.0.0.0 https://0.0.0.0 http://0.0.0.0:* https://0.0.0.0:* app://* file://* tauri://*] OLLAMA_RUNNERS_DIR: OLLAMA_SCHED_SPREAD:false OLLAMA_TMPDIR: ROCR_VISIBLE_DEVICES:]"
time=2024-07-22T13:17:50.145Z level=INFO source=images.go:784 msg="total blobs: 23"
time=2024-07-22T13:17:50.145Z level=INFO source=images.go:791 msg="total unused blobs removed: 0"
time=2024-07-22T13:17:50.145Z level=INFO source=routes.go:1147 msg="Listening on [::]:11434 (version 0.2.8-rc1)"
time=2024-07-22T13:17:50.145Z level=INFO source=payload.go:30 msg="extracting embedded files" dir=/tmp/ollama2936133361/runners
time=2024-07-22T13:17:51.862Z level=INFO source=payload.go:44 msg="Dynamic LLM libraries [cpu cpu_avx cpu_avx2 cuda_v11 rocm_v60102]"
time=2024-07-22T13:17:51.862Z level=INFO source=gpu.go:205 msg="looking for compatible GPUs"
time=2024-07-22T13:17:51.867Z level=INFO source=amd_linux.go:330 msg="amdgpu is supported" gpu=0 gpu_type=gfx1100
time=2024-07-22T13:17:51.868Z level=INFO source=amd_linux.go:330 msg="amdgpu is supported" gpu=1 gpu_type=gfx1100
time=2024-07-22T13:17:51.869Z level=INFO source=amd_linux.go:330 msg="amdgpu is supported" gpu=2 gpu_type=gfx1100
time=2024-07-22T13:17:51.869Z level=INFO source=amd_linux.go:259 msg="unsupported Radeon iGPU detected skipping" id=3 total="512.0 MiB"
time=2024-07-22T13:17:51.869Z level=INFO source=types.go:105 msg="inference compute" id=0 library=rocm compute=gfx1100 driver=6.7 name=1002:744c total="24.0 GiB" available="24.0 GiB"
time=2024-07-22T13:17:51.869Z level=INFO source=types.go:105 msg="inference compute" id=1 library=rocm compute=gfx1100 driver=6.7 name=1002:744c total="24.0 GiB" available="24.0 GiB"
time=2024-07-22T13:17:51.869Z level=INFO source=types.go:105 msg="inference compute" id=2 library=rocm compute=gfx1100 driver=6.7 name=1002:744c total="24.0 GiB" available="24.0 GiB"
[GIN] 2024/07/22 - 13:18:52 | 200 |      30.718µs |       127.0.0.1 | HEAD     "/"
[GIN] 2024/07/22 - 13:18:52 | 200 |     713.463µs |       127.0.0.1 | GET      "/api/tags"
[GIN] 2024/07/22 - 13:19:01 | 200 |      17.853µs |       127.0.0.1 | HEAD     "/"
[GIN] 2024/07/22 - 13:19:01 | 200 |    15.69545ms |       127.0.0.1 | POST     "/api/show"
time=2024-07-22T13:19:01.124Z level=INFO source=sched.go:717 msg="new model will fit in available VRAM, loading" model=/root/.ollama/models/blobs/sha256-a130d63aaf6f095d56914771afa0eb92ae04620ad496710957e9d650e4485b72 library=rocm parallel=4 required="45.3 GiB"
time=2024-07-22T13:19:01.125Z level=INFO source=memory.go:309 msg="offload to rocm" layers.requested=-1 layers.model=33 layers.offload=33 layers.split=11,11,11 memory.available="[24.0 GiB 24.0 GiB 24.0 GiB]" memory.required.full="45.3 GiB" memory.required.partial="45.3 GiB" memory.required.kv="1.0 GiB" memory.required.allocations="[15.4 GiB 15.4 GiB 14.4 GiB]" memory.weights.total="36.5 GiB" memory.weights.repeating="36.4 GiB" memory.weights.nonrepeating="102.6 MiB" memory.graph.full="1.3 GiB" memory.graph.partial="1.3 GiB"
time=2024-07-22T13:19:01.125Z level=INFO source=server.go:383 msg="starting llama server" cmd="/tmp/ollama2936133361/runners/rocm_v60102/ollama_llama_server --model /root/.ollama/models/blobs/sha256-a130d63aaf6f095d56914771afa0eb92ae04620ad496710957e9d650e4485b72 --ctx-size 8192 --batch-size 512 --embedding --log-disable --n-gpu-layers 33 --parallel 4 --tensor-split 11,11,11 --port 37323"
time=2024-07-22T13:19:01.125Z level=INFO source=sched.go:437 msg="loaded runners" count=1
time=2024-07-22T13:19:01.126Z level=INFO source=server.go:573 msg="waiting for llama runner to start responding"
time=2024-07-22T13:19:01.126Z level=INFO source=server.go:614 msg="waiting for server to become available" status="llm server error"
INFO [main] build info | build=1 commit="a8db2a9" tid="127915017311040" timestamp=1721654341
INFO [main] system info | n_threads=16 n_threads_batch=-1 system_info="AVX = 1 | AVX_VNNI = 0 | AVX2 = 0 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | AVX512_BF16 = 0 | FMA = 0 | NEON = 0 | SVE = 0 | ARM_FMA = 0 | F16C = 0 | FP16_VA = 0 | WASM_SIMD = 0 | BLAS = 1 | SSE3 = 1 | SSSE3 = 1 | VSX = 0 | MATMUL_INT8 = 0 | LLAMAFILE = 0 | " tid="127915017311040" timestamp=1721654341 total_threads=32
INFO [main] HTTP server listening | hostname="127.0.0.1" n_threads_http="31" port="37323" tid="127915017311040" timestamp=1721654341
llama_model_loader: loaded meta data with 26 key-value pairs and 995 tensors from /root/.ollama/models/blobs/sha256-a130d63aaf6f095d56914771afa0eb92ae04620ad496710957e9d650e4485b72 (version GGUF V3 (latest))
llama_model_loader: Dumping metadata keys/values. Note: KV overrides do not apply in this output.
llama_model_loader: - kv   0:                       general.architecture str              = llama
llama_model_loader: - kv   1:                               general.name str              = mistralai
llama_model_loader: - kv   2:                       llama.context_length u32              = 32768
llama_model_loader: - kv   3:                     llama.embedding_length u32              = 4096
llama_model_loader: - kv   4:                          llama.block_count u32              = 32
llama_model_loader: - kv   5:                  llama.feed_forward_length u32              = 14336
llama_model_loader: - kv   6:                 llama.rope.dimension_count u32              = 128
llama_model_loader: - kv   7:                 llama.attention.head_count u32              = 32
llama_model_loader: - kv   8:              llama.attention.head_count_kv u32              = 8
llama_model_loader: - kv   9:                         llama.expert_count u32              = 8
llama_model_loader: - kv  10:                    llama.expert_used_count u32              = 2
llama_model_loader: - kv  11:     llama.attention.layer_norm_rms_epsilon f32              = 0.000010
llama_model_loader: - kv  12:                       llama.rope.freq_base f32              = 1000000.000000
llama_model_loader: - kv  13:                          general.file_type u32              = 18
llama_model_loader: - kv  14:                       tokenizer.ggml.model str              = llama
llama_model_loader: - kv  15:                      tokenizer.ggml.tokens arr[str,32000]   = ["<unk>", "<s>", "</s>", "<0x00>", "<...
llama_model_loader: - kv  16:                      tokenizer.ggml.scores arr[f32,32000]   = [0.000000, 0.000000, 0.000000, 0.0000...
llama_model_loader: - kv  17:                  tokenizer.ggml.token_type arr[i32,32000]   = [2, 3, 3, 6, 6, 6, 6, 6, 6, 6, 6, 6, ...
llama_model_loader: - kv  18:                      tokenizer.ggml.merges arr[str,58980]   = ["▁ t", "i n", "e r", "▁ a", "h e...
llama_model_loader: - kv  19:                tokenizer.ggml.bos_token_id u32              = 1
llama_model_loader: - kv  20:                tokenizer.ggml.eos_token_id u32              = 2
llama_model_loader: - kv  21:            tokenizer.ggml.unknown_token_id u32              = 0
llama_model_loader: - kv  22:               tokenizer.ggml.add_bos_token bool             = true
llama_model_loader: - kv  23:               tokenizer.ggml.add_eos_token bool             = false
llama_model_loader: - kv  24:                    tokenizer.chat_template str              = {{ bos_token }}{% for message in mess...
llama_model_loader: - kv  25:               general.quantization_version u32              = 2
llama_model_loader: - type  f32:   65 tensors
llama_model_loader: - type  f16:   32 tensors
llama_model_loader: - type q8_0:   64 tensors
llama_model_loader: - type q6_K:  834 tensors
llm_load_vocab: special tokens cache size = 259
llm_load_vocab: token to piece cache size = 0.1637 MB
llm_load_print_meta: format           = GGUF V3 (latest)
llm_load_print_meta: arch             = llama
llm_load_print_meta: vocab type       = SPM
llm_load_print_meta: n_vocab          = 32000
llm_load_print_meta: n_merges         = 0
llm_load_print_meta: vocab_only       = 0
llm_load_print_meta: n_ctx_train      = 32768
llm_load_print_meta: n_embd           = 4096
llm_load_print_meta: n_layer          = 32
llm_load_print_meta: n_head           = 32
llm_load_print_meta: n_head_kv        = 8
llm_load_print_meta: n_rot            = 128
llm_load_print_meta: n_swa            = 0
llm_load_print_meta: n_embd_head_k    = 128
llm_load_print_meta: n_embd_head_v    = 128
llm_load_print_meta: n_gqa            = 4
llm_load_print_meta: n_embd_k_gqa     = 1024
llm_load_print_meta: n_embd_v_gqa     = 1024
llm_load_print_meta: f_norm_eps       = 0.0e+00
llm_load_print_meta: f_norm_rms_eps   = 1.0e-05
llm_load_print_meta: f_clamp_kqv      = 0.0e+00
llm_load_print_meta: f_max_alibi_bias = 0.0e+00
llm_load_print_meta: f_logit_scale    = 0.0e+00
llm_load_print_meta: n_ff             = 14336
llm_load_print_meta: n_expert         = 8
llm_load_print_meta: n_expert_used    = 2
llm_load_print_meta: causal attn      = 1
llm_load_print_meta: pooling type     = 0
llm_load_print_meta: rope type        = 0
llm_load_print_meta: rope scaling     = linear
llm_load_print_meta: freq_base_train  = 1000000.0
llm_load_print_meta: freq_scale_train = 1
llm_load_print_meta: n_ctx_orig_yarn  = 32768
llm_load_print_meta: rope_finetuned   = unknown
llm_load_print_meta: ssm_d_conv       = 0
llm_load_print_meta: ssm_d_inner      = 0
llm_load_print_meta: ssm_d_state      = 0
llm_load_print_meta: ssm_dt_rank      = 0
llm_load_print_meta: model type       = 8x7B
llm_load_print_meta: model ftype      = Q6_K
llm_load_print_meta: model params     = 46.70 B
llm_load_print_meta: model size       = 35.74 GiB (6.57 BPW) 
llm_load_print_meta: general.name     = mistralai
llm_load_print_meta: BOS token        = 1 '<s>'
llm_load_print_meta: EOS token        = 2 '</s>'
llm_load_print_meta: UNK token        = 0 '<unk>'
llm_load_print_meta: LF token         = 13 '<0x0A>'
llm_load_print_meta: max token length = 48
time=2024-07-22T13:19:01.377Z level=INFO source=server.go:614 msg="waiting for server to become available" status="llm server loading model"
ggml_cuda_init: GGML_CUDA_FORCE_MMQ:    no
ggml_cuda_init: GGML_CUDA_FORCE_CUBLAS: no
ggml_cuda_init: found 3 ROCm devices:
  Device 0: Radeon RX 7900 XTX, compute capability 11.0, VMM: no
  Device 1: Radeon RX 7900 XTX, compute capability 11.0, VMM: no
  Device 2: Radeon RX 7900 XTX, compute capability 11.0, VMM: no
llm_load_tensors: ggml ctx size =    1.53 MiB
time=2024-07-22T13:19:05.338Z level=INFO source=server.go:614 msg="waiting for server to become available" status="llm server not responding"
llm_load_tensors: offloading 32 repeating layers to GPU
llm_load_tensors: offloading non-repeating layers to GPU
llm_load_tensors: offloaded 33/33 layers to GPU
llm_load_tensors:      ROCm0 buffer size = 12510.78 MiB
llm_load_tensors:      ROCm1 buffer size = 12510.78 MiB
llm_load_tensors:      ROCm2 buffer size = 11475.99 MiB
llm_load_tensors:  ROCm_Host buffer size =   102.54 MiB
time=2024-07-22T13:19:05.590Z level=INFO source=server.go:614 msg="waiting for server to become available" status="llm server loading model"
time=2024-07-22T13:19:15.818Z level=INFO source=server.go:614 msg="waiting for server to become available" status="llm server not responding"
time=2024-07-22T13:19:16.646Z level=INFO source=server.go:614 msg="waiting for server to become available" status="llm server loading model"
llama_new_context_with_model: n_ctx      = 8192
llama_new_context_with_model: n_batch    = 512
llama_new_context_with_model: n_ubatch   = 512
llama_new_context_with_model: flash_attn = 0
llama_new_context_with_model: freq_base  = 1000000.0
llama_new_context_with_model: freq_scale = 1
time=2024-07-22T13:19:18.099Z level=INFO source=server.go:614 msg="waiting for server to become available" status="llm server not responding"
time=2024-07-22T13:19:18.912Z level=INFO source=server.go:614 msg="waiting for server to become available" status="llm server loading model"
llama_kv_cache_init:      ROCm0 KV buffer size =   352.00 MiB
llama_kv_cache_init:      ROCm1 KV buffer size =   352.00 MiB
llama_kv_cache_init:      ROCm2 KV buffer size =   320.00 MiB
llama_new_context_with_model: KV self size  = 1024.00 MiB, K (f16):  512.00 MiB, V (f16):  512.00 MiB
llama_new_context_with_model:  ROCm_Host  output buffer size =     0.55 MiB
llama_new_context_with_model: pipeline parallelism enabled (n_copies=4)
llama_new_context_with_model:      ROCm0 compute buffer size =   640.01 MiB
llama_new_context_with_model:      ROCm1 compute buffer size =   640.01 MiB
llama_new_context_with_model:      ROCm2 compute buffer size =   640.02 MiB
llama_new_context_with_model:  ROCm_Host compute buffer size =    72.02 MiB
llama_new_context_with_model: graph nodes  = 1510
llama_new_context_with_model: graph splits = 4
time=2024-07-22T13:19:20.722Z level=INFO source=server.go:614 msg="waiting for server to become available" status="llm server error"
time=2024-07-22T13:19:20.973Z level=ERROR source=sched.go:443 msg="error loading llama server" error="llama runner process has terminated: signal: segmentation fault (core dumped) "
[GIN] 2024/07/22 - 13:19:20 | 500 | 19.864840905s |       127.0.0.1 | POST     "/api/chat"
time=2024-07-22T13:19:25.973Z level=WARN source=sched.go:634 msg="gpu VRAM usage didn't recover within timeout" seconds=5.000268374 model=/root/.ollama/models/blobs/sha256-a130d63aaf6f095d56914771afa0eb92ae04620ad496710957e9d650e4485b72
time=2024-07-22T13:19:26.224Z level=WARN source=sched.go:634 msg="gpu VRAM usage didn't recover within timeout" seconds=5.250851195 model=/root/.ollama/models/blobs/sha256-a130d63aaf6f095d56914771afa0eb92ae04620ad496710957e9d650e4485b72
time=2024-07-22T13:19:26.473Z level=WARN source=sched.go:634 msg="gpu VRAM usage didn't recover within timeout" seconds=5.500499856 model=/root/.ollama/models/blobs/sha256-a130d63aaf6f095d56914771afa0eb92ae04620ad496710957e9d650e4485b72

```

### OS

Linux

### GPU

AMD

### CPU

AMD

### Ollama version

0.2.8-rc1-rocm

**Comment by dvelez3815** [comment]
On other version (1.44) is kinda okay but I got trash response...

```
2024/07/22 13:23:13 routes.go:1011: INFO server config env="map[OLLAMA_DEBUG:false OLLAMA_FLASH_ATTENTION:false OLLAMA_HOST:http://0.0.0.0:11434 OLLAMA_KEEP_ALIVE: OLLAMA_LLM_LIBRARY: OLLAMA_MAX_LOADED_MODELS:1 OLLAMA_MAX_QUEUE:512 OLLAMA_MAX_VRAM:0 OLLAMA_MODELS:/root/.ollama/models OLLAMA_NOHISTORY:false OLLAMA_NOPRUNE:false OLLAMA_NUM_PARALLEL:1 OLLAMA_ORIGINS:[http://localhost https://localhost http://localhost:* https://localhost:* http://127.0.0.1 https://127.0.0.1 http://127.0.0.1:* https://127.0.0.1:* http://0.0.0.0 https://0.0.0.0 http://0.0.0.0:* https://0.0.0.0:* app://* file://* tauri://*] OLLAMA_RUNNERS_DIR: OLLAMA_TMPDIR:]"
time=2024-07-22T13:23:13.315Z level=INFO source=images.go:725 msg="total blobs: 23"
time=2024-07-22T13:23:13.315Z level=INFO source=images.go:732 msg="total unused blobs removed: 0"
time=2024-07-22T13:23:13.316Z level=INFO source=routes.go:1057 msg="Listening on [::]:11434 (version 0.1.44)"
time=2024-07-22T13:23:13.316Z level=INFO source=payload.go:30 msg="extracting embedded files" dir=/tmp/ollama432537499/runners
time=2024-07-22T13:23:15.795Z level=INFO source=payload.go:44 msg="Dynamic LLM libraries [cuda_v11 rocm_v60002 cpu cpu_avx cpu_avx2]"
time=2024-07-22T13:23:15.800Z level=INFO source=amd_linux.go:301 msg="amdgpu is supported" gpu=0 gpu_type=gfx1100
time=2024-07-22T13:23:15.800Z level=INFO source=amd_linux.go:301 msg="amdgpu is supported" gpu=1 gpu_type=gfx1100
time=2024-07-22T13:23:15.800Z level=INFO source=amd_linux.go:301 msg="amdgpu is supported" gpu=2 gpu_type=gfx1100
time=2024-07-22T13:23:15.800Z level=INFO source=amd_linux.go:233 msg="unsupported Radeon iGPU detected skipping" id=3 total="512.0 MiB"
time=2024-07-22T13:23:15.800Z level=INFO source=types.go:71 msg="inference compute" id=0 library=rocm compute=gfx1100 driver=6.7 name=1002:744c total="24.0 GiB" available="24.0 GiB"
time=2024-07-22T13:23:15.800Z level=INFO source=types.go:71 msg="inference compute" id=1 library=rocm compute=gfx1100 driver=6.7 name=1002:744c total="24.0 GiB" available="24.0 GiB"
time=2024-07-22T13:23:15.800Z level=INFO source=types.go:71 msg="inference compute" id=2 library=rocm compute=gfx1100 driver=6.7 name=1002:744c total="24.0 GiB" available="24.0 GiB"
[GIN] 2024/07/22 - 13:23:23 | 200 |      25.789µs |       127.0.0.1 | HEAD     "/"
[GIN] 2024/07/22 - 13:23:23 | 200 |      557.66µs |       127.0.0.1 | GET      "/api/tags"
[GIN] 2024/07/22 - 13:23:30 | 200 |       21.14µs |       127.0.0.1 | HEAD     "/"
[GIN] 2024/07/22 - 13:23:30 | 200 |     643.581µs |       127.0.0.1 | POST     "/api/show"
[GIN] 2024/07/22 - 13:23:30 | 200 |     252.565µs |       127.0.0.1 | POST     "/api/show"
time=2024-07-22T13:23:30.273Z level=INFO source=amd_linux.go:301 msg="amdgpu is supported" gpu=0 gpu_type=gfx1100
time=2024-07-22T13:23:30.273Z level=INFO source=amd_linux.go:301 msg="amdgpu is supported" gpu=1 gpu_type=gfx1100
time=2024-07-22T13:23:30.273Z level=INFO source=amd_linux.go:301 msg="amdgpu is supported" gpu=2 gpu_type=gfx1100
time=2024-07-22T13:23:30.273Z level=INFO source=amd_linux.go:233 msg="unsupported Radeon iGPU detected skipping" id=3 total="512.0 MiB"
time=2024-07-22T13:23:30.493Z level=INFO source=memory.go:133 msg="offload to gpu" layers.requested=-1 layers.real=19 memory.available="24.0 GiB" memory.required.full="37.6 GiB" memory.required.partial="23.7 GiB" memory.required.kv="256.0 MiB" memory.weights.total="35.6 GiB" memory.weights.repeating="35.5 GiB" memory.weights.nonrepeating="102.6 MiB" memory.graph.full="184.0 MiB" memory.graph.partial="935.0 MiB"
time=2024-07-22T13:23:30.494Z level=INFO source=memory.go:133 msg="offload to gpu" layers.requested=-1 layers.real=19 memory.available="24.0 GiB" memory.required.full="37.6 GiB" memory.required.partial="23.7 GiB" memory.required.kv="256.0 MiB" memory.weights.total="35.6 GiB" memory.weights.repeating="35.5 GiB" memory.weights.nonrepeating="102.6 MiB" memory.graph.full="184.0 MiB" memory.graph.partial="935.0 MiB"
time=2024-07-22T13:23:30.495Z level=INFO source=memory.go:133 msg="offload to gpu" layers.requested=-1 layers.real=19 memory.available="24.0 GiB" memory.required.full="37.6 GiB" memory.required.partial="23.7 GiB" memory.required.kv="256.0 MiB" memory.weights.total="35.6 GiB" memory.weights.repeating="35.5 GiB" memory.weights.nonrepeating="102.6 MiB" memory.graph.full="184.0 MiB" memory.graph.partial="935.0 MiB"
time=2024-07-22T13:23:30.496Z level=INFO source=memory.go:133 msg="offload to gpu" layers.requested=-1 layers.real=33 memory.available="71.9 GiB" memory.required.full="38.0 GiB" memory.required.partial="38.0 GiB" memory.required.kv="256.0 MiB" memory.weights.total="35.6 GiB" memory.weights.repeating="35.5 GiB" memory.weights.nonrepeating="102.6 MiB" memory.graph.full="552.1 MiB" memory.graph.partial="2.7 GiB"
time=2024-07-22T13:23:30.497Z level=INFO source=memory.go:133 msg="offload to gpu" layers.requested=-1 layers.real=33 memory.available="71.9 GiB" memory.required.full="38.0 GiB" memory.required.partial="38.0 GiB" memory.required.kv="256.0 MiB" memory.weights.total="35.6 GiB" memory.weights.repeating="35.5 GiB" memory.weights.nonrepeating="102.6 MiB" memory.graph.full="552.1 MiB" memory.graph.partial="2.7 GiB"
time=2024-07-22T13:23:30.498Z level=INFO source=server.go:341 msg="starting llama server" cmd="/tmp/ollama432537499/runners/rocm_v60002/ollama_llama_server --model /root/.ollama/models/blobs/sha256-a130d63aaf6f095d56914771afa0eb92ae04620ad496710957e9d650e4485b72 --ctx-size 2048 --batch-size 512 --embedding --log-disable --n-gpu-layers 33 --parallel 1 --port 39509"
time=2024-07-22T13:23:30.498Z level=INFO source=sched.go:338 msg="loaded runners" count=1
time=2024-07-22T13:23:30.498Z level=INFO source=server.go:529 msg="waiting for llama runner to start responding"
time=2024-07-22T13:23:30.498Z level=INFO source=server.go:567 msg="waiting for server to become available" status="llm server error"
INFO [main] build info | build=1 commit="5921b8f" tid="136304919219264" timestamp=1721654610
INFO [main] system info | n_threads=16 n_threads_batch=-1 system_info="AVX = 1 | AVX_VNNI = 0 | AVX2 = 0 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | AVX512_BF16 = 0 | FMA = 0 | NEON = 0 | SVE = 0 | ARM_FMA = 0 | F16C = 0 | FP16_VA = 0 | WASM_SIMD = 0 | BLAS = 1 | SSE3 = 1 | SSSE3 = 1 | VSX = 0 | MATMUL_INT8 = 0 | LLAMAFILE = 1 | " tid="136304919219264" timestamp=1721654610 total_threads=32
INFO [main] HTTP server listening | hostname="127.0.0.1" n_threads_http="31" port="39509" tid="136304919219264" timestamp=1721654610
llama_model_loader: loaded meta data with 26 key-value pairs and 995 tensors from /root/.ollama/models/blobs/sha256-a130d63aaf6f095d56914771afa0eb92ae04620ad496710957e9d650e4485b72 (version GGUF V3 (latest))
llama_model_loader: Dumping metadata keys/values. Note: KV overrides do not apply in this output.
llama_model_loader: - kv   0:                       general.architecture str              = llama
llama_model_loader: - kv   1:                               general.name str              = mistralai
llama_model_loader: - kv   2:                       llama.context_length u32              = 32768
llama_model_loader: - kv   3:                     llama.embedding_length u32              = 4096
llama_model_loader: - kv   4:                          llama.block_count u32              = 32
llama_model_loader: - kv   5:                  llama.feed_forward_length u32              = 14336
llama_model_loader: - kv   6:                 llama.rope.dimension_count u32              = 128
llama_model_loader: - kv   7:                 llama.attention.head_count u32              = 32
llama_model_loader: - kv   8:              llama.attention.head_count_kv u32              = 8
llama_model_loader: - kv   9:                         llama.expert_count u32              = 8
llama_model_loader: - kv  10:                    llama.expert_used_count u32              = 2
llama_model_loader: - kv  11:     llama.attention.layer_norm_rms_epsilon f32              = 0.000010
llama_model_loader: - kv  12:                       llama.rope.freq_base f32              = 1000000.000000
llama_model_loader: - kv  13:                          general.file_type u32              = 18
llama_model_loader: - kv  14:                       tokenizer.ggml.model str              = llama
llama_model_loader: - kv  15:                      tokenizer.ggml.tokens arr[str,32000]   = ["<unk>", "<s>", "</s>", "<0x00>", "<...
llama_model_loader: - kv  16:                      tokenizer.ggml.scores arr[f32,32000]   = [0.000000, 0.000000, 0.000000, 0.0000...
llama_model_loader: - kv  17:                  tokenizer.ggml.token_type arr[i32,32000]   = [2, 3, 3, 6, 6, 6, 6, 6, 6, 6, 6, 6, ...
llama_model_loader: - kv  18:                      tokenizer.ggml.merges arr[str,58980]   = ["▁ t", "i n", "e r", "▁ a", "h e...
llama_model_loader: - kv  19:                tokenizer.ggml.bos_token_id u32              = 1
llama_model_loader: - kv  20:                tokenizer.ggml.eos_token_id u32              = 2
llama_model_loader: - kv  21:            tokenizer.ggml.unknown_token_id u32              = 0
llama_model_loader: - kv  22:               tokenizer.ggml.add_bos_token bool             = true
llama_model_loader: - kv  23:               tokenizer.ggml.add_eos_token bool             = false
llama_model_loader: - kv  24:                    tokenizer.chat_template str              = {{ bos_token }}{% for message in mess...
llama_model_loader: - kv  25:               general.quantization_version u32              = 2
llama_model_loader: - type  f32:   65 tensors
llama_model_loader: - type  f16:   32 tensors
llama_model_loader: - type q8_0:   64 tensors
llama_model_loader: - type q6_K:  834 tensors
llm_load_vocab: special tokens cache size = 259
llm_load_vocab: token to piece cache size = 0.3274 MB
llm_load_print_meta: format           = GGUF V3 (latest)
llm_load_print_meta: arch             = llama
llm_load_print_meta: vocab type       = SPM
llm_load_print_meta: n_vocab          = 32000
llm_load_print_meta: n_merges         = 0
llm_load_print_meta: n_ctx_train      = 32768
llm_load_print_meta: n_embd           = 4096
llm_load_print_meta: n_head           = 32
llm_load_print_meta: n_head_kv        = 8
llm_load_print_meta: n_layer          = 32
llm_load_print_meta: n_rot            = 128
llm_load_print_meta: n_embd_head_k    = 128
llm_load_print_meta: n_embd_head_v    = 128
llm_load_print_meta: n_gqa            = 4
llm_load_print_meta: n_embd_k_gqa     = 1024
llm_load_print_meta: n_embd_v_gqa     = 1024
llm_load_print_meta: f_norm_eps       = 0.0e+00
llm_load_print_meta: f_norm_rms_eps   = 1.0e-05
llm_load_print_meta: f_clamp_kqv      = 0.0e+00
llm_load_print_meta: f_max_alibi_bias = 0.0e+00
llm_load_print_meta: f_logit_scale    = 0.0e+00
llm_load_print_meta: n_ff             = 14336
llm_load_print_meta: n_expert         = 8
llm_load_print_meta: n_expert_used    = 2
llm_load_print_meta: causal attn      = 1
llm_load_print_meta: pooling type     = 0
llm_load_print_meta: rope type        = 0
llm_load_print_meta: rope scaling     = linear
llm_load_print_meta: freq_base_train  = 1000000.0
llm_load_print_meta: freq_scale_train = 1
llm_load_print_meta: n_yarn_orig_ctx  = 32768
llm_load_print_meta: rope_finetuned   = unknown
llm_load_print_meta: ssm_d_conv       = 0
llm_load_print_meta: ssm_d_inner      = 0
llm_load_print_meta: ssm_d_state      = 0
llm_load_print_meta: ssm_dt_rank      = 0
llm_load_print_meta: model type       = 8x7B
llm_load_print_meta: model ftype      = Q6_K
llm_load_print_meta: model params     = 46.70 B
llm_load_print_meta: model size       = 35.74 GiB (6.57 BPW) 
llm_load_print_meta: general.name     = mistralai
llm_load_print_meta: BOS token        = 1 '<s>'
llm_load_print_meta: EOS token        = 2 '</s>'
llm_load_print_meta: UNK token        = 0 '<unk>'
llm_load_print_meta: LF token         = 13 '<0x0A>'
time=2024-07-22T13:23:30.749Z level=INFO source=server.go:567 msg="waiting for server to become available" status="llm server loading model"
ggml_cuda_init: GGML_CUDA_FORCE_MMQ:   no
ggml_cuda_init: CUDA_USE_TENSOR_CORES: yes
ggml_cuda_init: found 3 ROCm devices:
  Device 0: Radeon RX 7900 XTX, compute capability 11.0, VMM: no
  Device 1: Radeon RX 7900 XTX, compute capability 11.0, VMM: no
  Device 2: Radeon RX 7900 XTX, compute capability 11.0, VMM: no
llm_load_tensors: ggml ctx size =    1.67 MiB
time=2024-07-22T13:23:34.710Z level=INFO source=server.go:567 msg="waiting for server to become available" status="llm server not responding"
llm_load_tensors: offloading 32 repeating layers to GPU
llm_load_tensors: offloading non-repeating layers to GPU
llm_load_tensors: offloaded 33/33 layers to GPU
llm_load_tensors:      ROCm0 buffer size = 12510.78 MiB
llm_load_tensors:      ROCm1 buffer size = 12510.78 MiB
llm_load_tensors:      ROCm2 buffer size = 11475.99 MiB
llm_load_tensors:  ROCm_Host buffer size =   102.54 MiB
time=2024-07-22T13:23:35.412Z level=INFO source=server.go:567 msg="waiting for server to become available" status="llm server loading model"
time=2024-07-22T13:23:45.648Z level=INFO source=server.go:567 msg="waiting for server to become available" status="llm server not responding"
time=2024-07-22T13:23:46.518Z level=INFO source=server.go:567 msg="waiting for server to become available" status="llm server loading model"
llama_new_context_with_model: n_ctx      = 2048
llama_new_context_with_model: n_batch    = 512
llama_new_context_with_model: n_ubatch   = 512
llama_new_context_with_model: flash_attn = 0
llama_new_context_with_model: freq_base  = 1000000.0
llama_new_context_with_model: freq_scale = 1
time=2024-07-22T13:23:47.972Z level=INFO source=server.go:567 msg="waiting for server to become available" status="llm server not responding"
time=2024-07-22T13:23:48.334Z level=INFO source=server.go:567 msg="waiting for server to become available" status="llm server loading model"
llama_kv_cache_init:      ROCm0 KV buffer size =    88.00 MiB
llama_kv_cache_init:      ROCm1 KV buffer size =    88.00 MiB
llama_kv_cache_init:      ROCm2 KV buffer size =    80.00 MiB
llama_new_context_with_model: KV self size  =  256.00 MiB, K (f16):  128.00 MiB, V (f16):  128.00 MiB
llama_new_context_with_model:  ROCm_Host  output buffer size =     0.14 MiB
llama_new_context_with_model: pipeline parallelism enabled (n_copies=4)
llama_new_context_with_model:      ROCm0 compute buffer size =   208.01 MiB
llama_new_context_with_model:      ROCm1 compute buffer size =   208.01 MiB
llama_new_context_with_model:      ROCm2 compute buffer size =   208.02 MiB
llama_new_context_with_model:  ROCm_Host compute buffer size =    24.02 MiB
llama_new_context_with_model: graph nodes  = 1510
llama_new_context_with_model: graph splits = 4
INFO [main] model loaded | tid="136304919219264" timestamp=1721654631
time=2024-07-22T13:23:51.845Z level=INFO source=server.go:572 msg="llama runner started in 21.35 seconds"
[GIN] 2024/07/22 - 13:23:51 | 200 | 21.576923759s |       127.0.0.1 | POST     "/api/chat"
```


chat:

```
>>> hi
###############################

>>> Send a message (/? for help)
``

**Comment by dhiltgen** [comment]
This appears to be a duplicate of #5708
